app/assets/javascripts/admin/components/admin-editable-field.js.es6

```diff
@@ -0,0 +1,23 @@
+export default Ember.Component.extend({
+  tagName: "",
+
+  buffer: "",
+  editing: false,
+
+  init() {
+    this._super();
+    this.set("editing", false);
+  },
+
+  actions: {
+    edit() {
+      this.set("buffer", this.get("value"));
+      this.toggleProperty("editing");
+    },
+
+    save() {
+      // Action has to toggle 'editing' property.
+      this.sendAction("action", this.get("buffer"));
```

Comment:
sendAction is getting deprecated in ember, try to use action closures please.

https://www.emberjs.com/deprecations/v3.x/#toc_ember-component-send-action